docs/data/base/components/switch/UnstyledSwitchIntroduction/tailwind/index.tsx

```diff
@@ -73,7 +73,7 @@ const Switch = React.forwardRef<HTMLSpanElement, SwitchProps>((props, ref) => {
           return {
             ...resolvedSlotProps,
             className: clsx(
-              `absolute block w-full h-full transition rounded-full border border-solid outline-none border-slate-300 dark:border-gray-700 shadow-[inset_0_1_1_rgb(0_0_0_/_0.05)] dark:shadow-[inset_0_1_1_rgb(0_0_0_/_0.5)] focus:ring-2 
+              `absolute block w-full h-full transition rounded-full border border-solid outline-none border-slate-300 dark:border-gray-700 shadow-[inset_0_1_1_rgb(0_0_0_/_0.05)] dark:shadow-[inset_0_1_1_rgb(0_0_0_/_0.5)] focus:shadow-purple-200 dark:focus:shadow-purple-600 
```

Comment:
@mnajdova — I've been trying to help Victor out with the focus state on the Switch Tailwind demo, but this simply doesn't work 😞 I tried various combinations, and even in different places (for the sake of debugging), and I can't tell why it's not picking up. Other styles were also not working, such as a hover state when the Switch is checked on (doing stuff like `hover:bg-purple-600` on line 79 down below has no effect).

Would love your help here to get this PR out of the door! 🙏

Reply:
Yes, let's merge the PR and open separate pull request for the different issues we found